scripts/send-restcmd.py

```diff
@@ -19,9 +22,25 @@
 
 args = parser.parse_args()
 
+app = Flask(__name__)
+
+@app.route('/', methods = ['POST'])
+def index():
+  json = request.get_json(force=True)
+  print("Result: ", json["result"])
+  print("of command: ", json["command"])
+  return 'Response received'
+
+def FlaskApp():
+  app.run(port=args.answer_port)
+
+if __name__ == "__main__":
+  flask_server = Process(target=FlaskApp)
```

Comment:
This single threaded mode is definitely fine for debugging, but we should check that we are not loosing responses.